notices.json

```diff
@@ -200,6 +200,44 @@
       }
     }
   },
+  {
+    "id": "bleve_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
```

Comment:
We can ship this notice now to give them advanced warnings. So we probably don't need this condition